src/main/java/seedu/address/model/tag/Tag.java

```diff
@@ -9,7 +9,8 @@
  */
 public class Tag {
 
-    public static final String MESSAGE_CONSTRAINTS = "Tags names should be alphanumeric";
+    public static final String MESSAGE_CONSTRAINTS = "Tags names should be alphanumeric.\n"
+            + "If you want to not specify the tag, leave out t/TAG entirely.";
```

Comment:
Shouldn't this say "nonempty and alphanumeric" then? But what about `edit` command...

Reply:
That's right, but I'm afraid that "nonempty" could be misunderstood as we can't have no tag by users.

Reply:
Yes, but seeing "tag names should be alphanumeric" for a command `edit 1 t/ t/x` is extremely confusing.

Reply:
Hmm, then how can I inform them how to not specify it when they add this...